Formula/jellyfish.rb

```diff
@@ -0,0 +1,24 @@
+class Jellyfish < Formula
+  # cite Marçais_2001: "https://doi.org/10.1093/bioinformatics/btr011"
+  desc "Fast, memory-efficient counting of DNA k-mers"
+  homepage "http://www.genome.umd.edu/jellyfish.html"
+  url "https://github.com/gmarcais/Jellyfish/releases/download/v2.2.9/jellyfish-2.2.9.tar.gz"
+  sha256 "854511d06f04cc2a8459d3f171862381d98430494194ff6f1970d1ffb74c80a8"
+
+  depends_on "gcc" if OS.mac? # Fix error: use of undeclared identifier 'aligned_alloc'
+  depends_on "zlib" unless OS.mac?
+    
+  fails_with :clang # Fix error: use of undeclared identifier 'aligned_alloc'
+
+  needs :cxx11
```

Comment:
I usually put `fails_with` and `needs` above `depends_on`. Does `brew audit` have a preference?

Reply:
Brew audit doesn't care. I've been putting `needs` before `install` as that is where it seemed to be in previous formula.

Reply:
I'm fine with either.